lesson18_task3.py

```diff
@@ -0,0 +1,70 @@
+"""
+Write a class TypeDecorators which has several methods for converting results of functions to a specified type
+(if it's possible):
+"""
+
+from functools import wraps
+from typing import Callable
+
+
+class TypeDecorators:
+    @staticmethod
+    def to_int(func: Callable[[str], str]) -> Callable[[str], int]:
```

Comment:
Very nice that you even used Callable parameters and returning result type hinting.